ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -392,7 +392,11 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
                                                                               text: text, labelAccessoryViewMarginLeading: labelAccessoryViewMarginLeading)
 
         let availableWidth = textAreaWidth - (leadingAccessoryAreaWidth + trailingAccessoryAreaWidth)
-        return text.preferredSize(for: font, width: availableWidth, numberOfLines: numberOfLines)
+
+        // The width from preferredSize() will hug the label, which results in cut-off text if the label has additional attributes.
+        // We want to have the available width to prevent any cut-off.
+        return CGSize(width: text.isEmpty ? 0 : availableWidth,
+                      height: text.preferredSize(for: font, width: availableWidth, numberOfLines: numberOfLines).height)
```

Comment:
if the label has attributed text it might be worth calculating with its attributed string with open func size() -> CGSize ?